drivers/veteran_confirmation/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -0,0 +1,43 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module VeteranConfirmation::GrdaWarehouse
+end
+
+module VeteranConfirmation::GrdaWarehouse::Hud
+  module ClientExtension
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :va_check_histories, class_name: 'VeteranConfirmation::VaCheckHistory'
+      has_one :recent_va_check, -> do
+        merge(VeteranConfirmation::VaCheckHistory.recent)
```

Comment:
```suggestion
        merge(VeteranConfirmation::VaCheckHistory.recent(MIN_DAYS))
```

Reply:
I took a different take on this as I realized I needed different information in different contexts.